core/iwasm/fast-jit/fe/jit_emit_function.c

```diff
@@ -8,6 +8,7 @@
 #include "../jit_frontend.h"
 #include "../jit_codegen.h"
 #include "../../interpreter/wasm_runtime.h"
+#include "wasm.h"
```

Comment:
Do we need to include wasm.h? Including wasm_runtime.h should already include wasm.h.
If yes, had better change to `../../interpreter/wasm.h`